dts/bindings/sensor/st,lis2mdl-i2c.yaml

```diff
@@ -17,3 +17,15 @@ properties:
         This pin defaults to active high when produced by the sensor.
         The property value should ensure the flags properly describe
         the signal that is presented to the driver.
+
+    single-mode:
+      type: boolean
+      description: |
+        Set to config the sensor in single measurement mode. Leave
+        unset to configure the sensor in continuous measurement mode.
+
+    cancel-offset:
+      type: boolean
+      description: |
+        Set to enable the offset cancellation. Otherwise it would be
+        disabled as default.
```

Comment:
Why did you change only i2c and not spi binding?

Reply:
Ah! And here it would be nice to have a common st,lis2mdl-common.yaml file, to be shared between i2c and spi. You can see that there are currently already other examples.

Reply:
This part seems addressed correctly now.